Simplenote/SPAppDelegate.m

```diff
@@ -496,6 +496,11 @@ - (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDiction
         return YES;
     }
 
+    // Dismiss sidebar if visible
+    if (self.sidebarViewController.isSidebarVisible) {
+        [self.sidebarViewController hideSidebarWithAnimation:NO];
+    }
```

Comment:
`popToNoteList` already calls `sidebarViewController.hideSidebar(withAnimation: animated)`, do we need to dismiss sidebar here as well?

Reply:
Ah no we don't, that was a POC for me to see if this would work.  Must have gotten committed accidentally